src/burner/libretro/retro_input.cpp

```diff
@@ -1581,6 +1581,30 @@ static INT32 GameInpSpecialOne(struct GameInp* pgi, INT32 nPlayer, char* szb, ch
 		}
 	}
 
+	// Defender
+	// Stargate
+	// Robotron
+	// Joust
+	// Sinistar
+	if ((parentrom && strcmp(parent, "defender") == 0) ||
+	  (drvname && strcmp(drvname, "defender") == 0) ||
+		(parentrom && strcmp(parentrom, "stargate") == 0) ||
+		(drvname && strpcmp(drvname, "stargate") == 0) ||
```

Comment:
Another typo, `strpcmp` should be `strcmp`

Reply:
Fixed